src/components/NFTOriginVerifier.tsx

```diff
@@ -206,15 +272,20 @@ export default function NFTOriginVerifier() {
     lookupChain,
     lookupAsset,
     lookupTokenId,
+    lookupCreatorAddress,
+    lookupCollectionName,
+    lookupTokenName,
   ]);
   const readableAddress =
     originInfo &&
     originInfo.chainId &&
     originInfo.assetAddress &&
-    hexToNativeAssetString(
+    (hexToNativeAssetString(
```

Comment:
I think this validation could be a `const` with a semantic var name on top of the scope.

Reply:
fixed